deps/wazuh_testing/wazuh_testing/tools/agent_simulator.py

```diff
@@ -869,3 +869,36 @@ def create_agents(agents_number, manager_address, cypher, fim_eps=None, authd_pa
                                 os=agent_os, version=agent_version))
         agent_count = agent_count + 1
     return agents
+
+
+def send_ping_pong_messages(protocol, manager_address, port):
+    """This function sends the ping message to the manager
+
+    This message is the first of many between the manager and the agents. It is used to check if both of them are ready
+    to send and receive other messages
+
+    Args:
+        protocol (str): it can be UDP or TCP
+        manager_address (str): address of the manager. IP and hostname are valid options
+        port (int): port where the manager has bound the remoted port
+
+    Returns:
+        bytes: returns the #pong message from the manager
+
+    Raises:
+        ConnectionRefusedError: if there's a problem while sending messages to the manager
+    """
+    if protocol == "UDP":
```

Comment:
I think that in all other cases, tcp and udp are used in lower case. We should create a constant variable to model the format of this string